### PR TITLE
Centralized API for Feedback mesasges

### LINKED ISSUE #196: Refactor: unify feedback messages
**Is your feature request related to a problem? Please describe.**
Although in our designs we have a single way to pass success failure feedback, on the implementation we don't. We should unify feedback messages

**Describe the solution you'd like**
Add a new key in the shared map with messages, allowing us to configure if it is success or failure. Use our default designs: https://www.figma.com/file/oZJUIyHRmLyLIKDxYebvLg/Form-elements?node-id=7%3A296

### PR DESCRIPTION
**Pull request by RodrigoLeiteF**
## What issue does this PR close
Closes #196

### CODE REVIEW

File: src/elm/Session/LoggedIn.elm [hunk 1]
```diff
@@ -204,6 +218,24 @@ view thisMsg page ({ shared } as model) content =
...
+        , style "background-color" (getBackgroundColor feedback.success)
+        ]
+        [ span [ class "ml-auto invisible" ] []
+        , span [ class "flex items-center text-sm h-10 leading-snug text-white font-bold" ]
+            [ text feedback.message ]
+        , span
+            [ class "ml-auto text-lg text-white font-bold mr-5 cursor-pointer"
+            , onClick HideFeedback
+            ]
+            [ text "X"
```
Comment (lucca65) [inline src/elm/Session/LoggedIn.elm:234]: Its best we use the `x` icon so we are consistent with the rest of the application.

We can have

```elm
Icons.close ""
```

File: src/elm/Main.elm [hunk 2]
```diff
@@ -488,6 +488,16 @@ updateLoggedInUResult toStatus toMsg model uResult =
 
                         _ ->
                             ( m, cmds_ )
+
+                LoggedIn.ShowFeedback x ->
```
Comment (lucca65) [inline src/elm/Main.elm:492]: we should have a better name for this value

File: src/elm/Session/LoggedIn.elm [hunk 3]
```diff
@@ -478,6 +518,15 @@ viewFooter _ =
         ]
 
 
+getBackgroundColor : Bool -> String
+getBackgroundColor isSuccess =
+    if isSuccess == True then
+        "#8ACC9E"
+
+    else
+        "#DB1B1B"
```
Comment (lucca65) [inline src/elm/Session/LoggedIn.elm:527]: we should use tailwind classes for that `green` and `red`. Actually maybe we don't need a function to encapsulate this `if`

Comment (RodrigoLeiteF) [reply src/elm/Session/LoggedIn.elm:527]: I tried those classes but for some reason they wouldn't work. Maybe we're using an outdated version?

**Pull request merged**